pkg/util/tas/tas.go

```diff
@@ -30,6 +30,30 @@ func DomainID(levelValues []string) TopologyDomainID {
 	return TopologyDomainID(strings.Join(levelValues, ","))
 }
 
+func IsTAS(pod *corev1.Pod) bool {
+	if IsExplicitTAS(pod.Annotations) {
+		return true
+	}
+	if _, ok := pod.Annotations[kueuealpha.PodSetUnconstrainedTopologyAnnotation]; ok {
+		return true
+	}
+	// TODO: remove after 0.16
+	if _, ok := pod.Labels[kueuealpha.TASLabel]; ok {
+		return true
+	}
+	return false
+}
+
+func IsExplicitTAS(annots map[string]string) bool {
+	if _, ok := annots[kueuealpha.PodSetPreferredTopologyAnnotation]; ok {
+		return true
+	}
+	if _, ok := annots[kueuealpha.PodSetRequiredTopologyAnnotation]; ok {
+		return true
+	}
```

Comment:
```suggestion
	if _, ok := annots[kueue.PodSetPreferredTopologyAnnotation]; ok {
		return true
	}
	if _, ok := annots[kueue.PodSetRequiredTopologyAnnotation]; ok {
		return true
	}
```

Shouldn't we use `kueue` as import name since Topology is already beta API?

Reply:
cc @mbobrovskyi

Reply:
Done